### DISCUSSION TITLE
[Announcement] Final exam is graded & claim session

### DISCUSSION THREAD

**Discussion by sujin0529** [prosyslab-classroom/cs348-information-security#393] [Announcements]
Hi all,

The final exam is graded.

You can check your score and criteria for each problem at the [Gradescope](https://www.gradescope.com/courses/516803).

If you have any claim, contact us until **next Tuesday(06.20), 14:00** via [cs348.ta@prosys.kaist.ac.kr](mailto:cs348.ta@prosys.kaist.ac.kr) or meet in person.
(+) Also, you can use "Regrade request(재채점 요청)" in Gradescope system.

FYI, here is the statistic for this exam.
![image](https://github.com/prosyslab-classroom/cs348-information-security/assets/44044134/d96868e5-2c2c-4e53-9df5-da8c7754156f)

Thanks.

**Comment by m-spitfire** [comment]
Hello. When and where can I meet you in person to do the claim? Thanks.

**Comment by yeonhee-ryou** [comment]
@m-spitfire mail us with the problem list you want to check
